data/console/test_java_transactional-server.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Variables Definition
+# --------------------
+PWD=$(/usr/bin/pwd)
+DIR="/var/tmp"
```

Comment:
The only differences between this and the original [test_java.sh](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/console/test_java.sh#L6) is that you are removing all the java tests.

Acoording to the [documentation](https://documentation.suse.com/sles/15-SP2/html/SLES-all/cha-transactional-updates.html#sec-install-tu), `/var/` should be fine here... if there's a problem somehow, it might be a product issue instead.

Reply:
it failed already with: http://10.162.23.47/tests/8473#step/java/43
it tries to remove files from /usr/lib64

Any other idea?

Reply:
For now you could work around this by only testing the default java version, i.e. not doing any call to `update-alternatives`. It might also work to just ignore java 1.8 and still test newer ones, as only versions of java which don't provide the paths in the error message break.

Reply:
yes, thanks!
Please review changes for this and my test run:
http://10.162.23.47/tests/8525#step/java/35

Reply:
The test is looking good, but can you merge this into `test_java.sh` by just skipping the `update-alternatives` calls if it's a transactional server?

Reply:
the problem is shell script itself, can we check something lile get_var in shell script? 
so the first idea can be: modify script after wget. but I think this would be little complex and this is not good for maintance in future. Please let me know about this, thanks.
I suggest to create a new ticket for issue with test_java.sh for non-transactional-server if we want to fix something on whatever.

Reply:
You could either  pass some flag to the script, e.g. `./test_java.sh --transactional-server` or just do `findmnt / | grep -q ',ro'`

Reply:
findmnt could be indeed an good way to check for transactional-server in script. I compared this differences between both test_java files. I think for now this is easier and better to use different scripts. Again in this PR I want schedule java on transactional server and make less changes on script. This is also ticket about. We can fix other issue later. I have created a ticket: https://progress.opensuse.org/issues/77716
Please feel free to check it and modify requirements for issues which has to do with java in general.

The idea about passing flag to script sounds good, but I am not sure about this in details. I need check this as well, Thanks for your input!

Reply:
> Again in this PR I want schedule java on transactional server and make less changes on script. This is also ticket about. We can fix other issue later.

If it's already done properly in the first PR, there's no followup work required. If this is merged and it doesn't immediately create an issue, I'm pretty sure that the code won't be touched for cleanup or other maintenance reasons in the next months at all - that's the reality of most software development. I'm trying to avoid that.

Reply:
okay, I merged test_java.sh so far. Please check http://10.162.23.47/tests/8565#step/java/35 
Please let me know if something important is missing, thanks.

Reply:
updated now.
please check:
http://10.162.23.47/tests/8567#step/java/35 (transactional server)
http://10.162.23.47/tests/8568#step/java/24 (extra_test_testmode sles)
http://10.162.23.47/tests/8570#step/java/18 (extra_test_testmode tw)
http://10.162.23.47/tests/8569#step/java/21 (jeos)